### DISCUSSION TITLE
🚀 New flaresolverr image with improved detection and nodriver support

### DISCUSSION THREAD

**Discussion by svaningelgem** [FlareSolverr/FlareSolverr#1416] [General]
# 🚀 New flaresolverr docker image with improved detection and nodriver support

Advantages:
- Unlike the official image, this one works on Windows hosts (it fixes the "user flaresolverr not found" error).
- LANG is by default "en" (not your systems one)
- Using the "shadow" trick from @alexandervlk that improves detection
- Using the "nodriver" driver from @21hsmw 

You can try it out using:

```shell
docker run -d --name flaresolverr --restart unless-stopped -p 8191:8191 ghcr.io/svaningelgem/flaresolverr:latest
```

This combines the best of both approaches to provide better reliability and detection capabilities.

**Comment by IVazharov** [comment]
Do you have plans for making a non-Docker Windows build as well?

**Reply by svaningelgem** [reply]
Just check out the branch and run it? I'll have a look on how to create an EXE, but I would prefer this project adds the capabilities that are present in the community already and release it that way.

**Comment by cplieger** [comment]
Seems to no longer work, getting "172.16.16.1 POST http://192.168.1.77:18191/v1 500 Internal Server Error" for multiple websites.

Only image that still works is: alexfozor/flaresolverr:pr-1300-experimental

**Reply by svaningelgem** [reply]
Yeah, I was investigating what was going on, but couldn't figure it out.
On my windows machine, running from the code, my latest version works just fine...
But when dockerized (ie: Linux), the exact same code couldn't.

Instead of spending more time on it, I'll be using the one from Alex as mentioned above

If you want to run that too:

```
docker stop flaresolverr
docker rm flaresolverr
docker run -d \
  --name flaresolverr \
  --restart=always \
  -p 127.0.0.1:8191:8191 \
  alexfozor/flaresolverr:pr-1300-experimental
```

**Reply by TapeMeOh** [reply]
I can't seem to get either of them to work.

Just gets stuck at launching web browser on my Windows 10 system.

> 
> FlareSolverr 3.3.21
> >> 2025-03-06 12:37:48 INFO     Testing web browser installation...
> >> 2025-03-06 12:37:48 INFO     Platform: Windows-10-10.0.19045-SP0
> >> 2025-03-06 12:37:48 INFO     Chrome / Chromium path: C:\Program Files\Google\Chrome\Application\chrome.exe
> >> 2025-03-06 12:37:48 INFO     Chrome / Chromium major version: 133
> >> 2025-03-06 12:37:48 INFO     Launching web browser...
